### PR TITLE
Add Kyocera Mac Driver 5.0

### PR DESCRIPTION
**Pull request by krystof-k**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask-drivers/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask-drivers/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by krystof-k** [comment]
Hopefully it will be fine. I'm not sure about the name, but there is nothing else from Kyocera in any tap, so it should not be confusing to stick just with `kyocera`.

**Comment by vitorgalvao** [comment]
> I'm not sure about the name, but there is nothing else from Kyocera in any tap

Still, it would be helpful to have a little more. This is a driver for what?

**Comment by krystof-k** [comment]
I assume for all Kyocera printers. So something like `kyocera-printer-drivers` would be fine?

File: Casks/kyocera.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cask 'kyocera' do
+  version '2019.11.21'
+  sha256 'a0a9ebbeb577a32c389f20bcd11af2b8094d04fa8acbe8cc73de444b102d0a4e'
+
+  url "https://www.kyoceradocumentsolutions.eu/content/download-center/eu/drivers/all/MacPhase50_#{version.dots_to_underscores}_zip.download.zip"
+  name 'Kyocera Mac Driver 5.0'
```
Comment (vitorgalvao) [inline Casks/kyocera.rb]: Versions should not be included in `name`.

**Comment by vitorgalvao** [comment]
> So something like `kyocera-printer-drivers` would be fine?

Better. Can you guide me to `url` from the `homepage`?

File: Casks/kyocera.rb (hunk 1 above)
Comment (krystof-k) [reply Casks/kyocera.rb]: Well, it's not exactly a version, the version is the datestamp, but I can remove it.

Comment (vitorgalvao) [reply Casks/kyocera.rb]: Should still be removed, yes.

**Comment by krystof-k** [comment]
That is hard. You have to pick a [specific printer](https://dlc.kyoceradocumentsolutions.eu/index/service/dlc.false.utility.TASKALFA5053CI._.EN.html).

**Comment by vitorgalvao** [comment]
> That is hard. You have to pick a [specific printer](https://dlc.kyoceradocumentsolutions.eu/index/service/dlc.false.utility.TASKALFA5053CI._.EN.html).

Any is fine. Alternatively, if you can find an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md), that also works.

**Comment by vitorgalvao** [comment]
Great work. And fast, too! Let’s wait for CI and merge.

Thank you.

**Pull request merged**